flocker/control/schema/endpoints.yml

```diff
@@ -74,3 +74,12 @@ definitions:
       # more about what it means to not have a primary manifestation.
       - primary
     additionalProperties: false
+
+  # A sequence of datasets
+  datasets_array:
+    type: array
+    items:
+      description: "The dataset"
+      type: object
+      oneOf:
+        - {"$ref": "#/definitions/datasets" }
```

Comment:
Add a note and an issue about how config/datasets and state/datasets may not be the same. 

See https://github.com/ClusterHQ/flocker/pull/1101#discussion_r23784531 and https://clusterhq.atlassian.net/browse/FLOC-1247